smarts/core/scenario.py

```diff
@@ -22,27 +22,32 @@
 import logging
 import math
 import os
-import sys
 import pickle
 import random
 import uuid
-
-import numpy as np
-
-from dataclasses import dataclass, field
+from dataclasses import dataclass
+from dataclasses import field
 from functools import lru_cache
-from itertools import cycle, product
+from itertools import cycle
+from itertools import product
 from pathlib import Path
-from typing import Any, Dict, Sequence, Tuple
+from typing import Any
+from typing import Dict
+from typing import Sequence
+from typing import Tuple
```

Comment:
why are we spreading out these imports across multiple lines?

Reply:
I used a formatter and the justification for it is that it allows for easier navigation of merges.

Reply:
Wait, you used a formatter other than black! Heathen!!

In all seriousness, I do prefer the old format of grouping all modules from the same parent on one line.

Reply:
Fair enough.